packages/babel-parser/test/fixtures/typescript/types/conditional-infer/options.json

```diff
@@ -1,5 +1,4 @@
 {
   "plugins": ["typescript"],
-  "createParenthesizedExpressions": true,
```

Comment:
These tests are copied as new `*-parenthesized-expressions` tests, as in Babel 8 this option defaults to false.